app/controllers/credentials/worldpay.controller.js

```diff
@@ -35,12 +35,26 @@ async function updateWorldpayCredentials (req, res, next) {
     const credential = getCredentialByExternalId(req.account, req.params.credentialId)
     const results = credentialsForm.validate(req.body)
 
+    if (!results.errorSummaryList.length) {
+      const merchantId = req.body.merchantId
+      if (req.account.allow_moto && !merchantId.endsWith('MOTO')) {
+        results.errors.merchantId = 'Enter a valid MOTO merchant account code. MOTO payments are enabled for the account'
```

Comment:
I think just ‘merchant code’ would be better because it matches the terminology we use on the page (and what Worldpay use). Not sure the word ‘valid’ is really adding much.